TCMSApp/src/client/app/runs/runs.service.js

```diff
@@ -8,8 +8,16 @@
     RunsApiService.$inject = ['$resource', 'apiUrl'];
 
     function RunsApiService($resource, apiUrl) {
+        var currentRun;
 
         return {
+            currentRun: function(_currentRun) {
+                if (!!_currentRun) {
```

Comment:
if you're checking that no argument is sent the best way is to write `typeof _currentRun === 'undefined'`